main.go

```diff
@@ -158,6 +158,7 @@ func main() {
 		[]*types.NetworkIdentifier{cfg.Network},
 		nil,
 		services.MempoolCoins,
+		``,
```

Comment:
Is this similar to empty string?

Reply:
yes, but it seems backtick in Go stands for raw string. Changed to double quote now